### ISSUE TITLE
Support for radio stations

### ISSUE THREAD

**Issue by Znurre** [Znurre/Subsoniq#3]
Add support for listing, adding, removing and listening to radio stations.

**Comment by KingDuckZ** [comment]
Is there any progress on this?

**Comment by Znurre** [comment]
Nope. My first priority is finishing the persistent playlist support, but seeing as this feature has been requested a few times now I will make sure to prioritize it.
